### DISCUSSION TITLE
Downgrade Quasar CLI NPM Commands

### DISCUSSION THREAD

**Discussion by susansiow** [quasarframework/quasar#13277] [CLI - BEX mode]
According to [this thread](https://forum.quasar-framework.org/topic/1836/downgrade-v0-15-to-v0-14/10?_=1651117716190&lang=en-US), Quasar CLI can be downgraded with these commands > `npm uninstall -g quasar-cli; npm install -g quasar-cli@[version number]`. If fact, is running one command sufficient - `npm install -g quasar-cli@[version number]` or both commands is better?

**Comment by MilosPaunovic** [comment]
`npm install -g whatever` will install the latest version of the package globally, so it's sufficient to run that command to have latest global package on your machine.

**Reply by susansiow** [reply]
Sorry, I mean which is the better method to downgrade Quasar CLI, just in case I need to do so for other Quasar projects on my local computer.

1. `npm uninstall -g quasar-cli; npm install -g quasar-cli@[version number]` (run the commands to uninstall, then install the specific older version of Quasar CLI)
2. `npm install -g quasar-cli@[version number]` (just run the command to downgrade to a specific older version of Quasar CLI)

No. 1 is better because this method is cleaner?
